docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -120,7 +117,7 @@ In the `pom.xml` file, extend the `native` profile with:
 </plugin>
 ----
 
-Then, create the `src/test/java/org/acme/quickstart/GreetingResourceIT.java` with the following content:
+You'll also find the `src/test/java/org/acme/quickstart/NativeGreetingResourceIT.java` test class in the project:
```

Comment:
It's not a good idea to change that as the code of the quickstart has it named GreetingResourceIT (and it's the same for all the other quickstarts)